site/docs/components/toggle-button/accessibility.mdx

```diff
@@ -36,13 +36,13 @@ data:
 - If focus is below the toggle button group, move focus into the toggle button group. Focus is set to the active button.
 
 </KeyboardControl>
-<KeyboardControl keyOrCombos="ArrowRight / ArrowDown">
+<KeyboardControl keyOrCombos="→ / ↓">
```

Comment:
@mikearildbrown what do you think of this change? can revert it. Although I feel at least "Right Arrow" is more clear to read than "ArrowUp".